client/src/Hooks/useReactionsFetch.js

```diff
@@ -0,0 +1,15 @@
+import { useState, useEffect } from 'react';
+import api from '../api.js';
+export default function useReactionsFetch () {
+	const [reactions, setReactions] = useState({});
+
+	useEffect(() => {
+		api.sendReactionsRequest().then((res) => {
+			setReactions(res.data);
+		}).catch((error) =>
+			console.log(error)
+		);
+	}, []);
+
+	return reactions;
```

Comment:
This hook should return the error as well for you to show it somehow in UI later (maybe in a toast).

Reply:
I would recommend to return `[error, reactions]` or `{error, reactions}`.
So you can use this hook like:
`const [error, reactions] = useReactionsFetch()` or
`const { error, reactions } = useReactionsFetch()`